Docker/test/curl/test_curl.sh

```diff
@@ -5,17 +5,24 @@ check_if_up() {
     PATH=$2
 
     FULL_URL="$1$2"
+    DISPLAY_URL="http://$FULL_URL"
 
     if [ -z "$URL" ]; then
+        echo "❌ check_if_up called with an empty URL (with PATH: $2)"
+
         return
     fi
 
-    if /usr/bin/curl --fail --retry 120 --retry-all-errors --retry-delay 1 --max-time 10 --retry-max-time 120 --show-error --output /dev/null --silent "$FULL_URL"; then
-        echo "Successfully loaded $URL!"
-    else
-        echo "Could not retrieve $URL"
-        exit 1
+    if ! (/usr/bin/curl --fail --output /dev/null --silent "$FULL_URL";) then
+        echo "🔄 Waiting for $DISPLAY_URL"
+
+        if ! (/usr/bin/curl --fail --retry 120 --retry-all-errors --retry-delay 1 --max-time 10 --retry-max-time 120 --output /dev/null --silent "$FULL_URL";) then
+            echo "❌ Could not load $DISPLAY_URL"
+            exit 1
+        fi
     fi
+
+    echo "✅ Successfully loaded $DISPLAY_URL!"
```

Comment:
Found it a bit distracting to have the ! at the end of the url, works without as well, I guess. Love the emojis btw :)

```suggestion
    echo "✅ Successfully loaded $DISPLAY_URL"
```

Reply:
Agreed. I thnk the "!" was just carried over from the original implementation.